chains/substrate/events/events.go

```diff
@@ -4,22 +4,72 @@
 package events
 
 import (
-	"github.com/ChainSafe/chainbridge-core/relayer/message"
-	core_types "github.com/ChainSafe/chainbridge-core/types"
 	"github.com/centrifuge/go-substrate-rpc-client/v4/types"
 )
 
-type Deposit struct {
-	DestinationDomainID uint8
-	ResourceID          core_types.ResourceID
-	DepositNonce        uint64
-	DepositType         message.TransferType
-	SenderAddress       types.AccountID
-	Data                []byte
-	HandlerResponse     []byte
-}
-
 type Events struct {
 	types.EventRecords
-	Deposit []Deposit
+	SygmaBridge_Deposit         []EventDeposit
+	SygmaBasicFeeHandler_FeeSet []EventFeeSet
+
+	SygmaBridge_ProposalExecution      []EventProposalExecution
+	SygmaBridge_FailedHandlerExecution []EventFailedHandlerExecution
+	SygmaBridge_Retry                  []EventRetry
+	SygmaBridge_BridgePaused           []EventBridgePaused
+	SygmaBridge_BridgeUnpaused         []EventBridgeUnpaused
+}
+type EventDeposit struct {
+	Phase        types.Phase
+	DestDomainID types.U8
+	ResourceID   types.Bytes32
+	DepositNonce types.U64
+	Sender       types.AccountID
+	TransferType [1]byte
+	CallData     []byte
+	Handler      [1]byte
+	Topics       []types.Hash
+}
+
+type EventFeeSet struct {
```

Comment:
Kinda redudant to have events.EventFeeSet it can just be events.FeeSet.
To applies to all.

Reply:
Should i rename them all andremove the Event prefix?

Reply:
Would be nice